.github/workflows/python-tests.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     with:
       platform: centos7
       usage: opensearch
+      build_ref: ${github.ref}
```

Comment:
I dont think we need this. The workflow runs on respective branches. So we commit or backport to 2.x, the workflows will run on 2.x not on main.

Reply:
It is designed specifically in this way so other repos can reuse this yaml:
https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/get-ci-image-tag.yml#L31-L36